tests/unit/test_db_sync.py

```diff
@@ -72,48 +73,83 @@ def test_column_type_mapping(self):
         json_arr = {"type": ["array"]}
 
         # Mapping from JSON schema types ot Snowflake column types
-        self.assertEquals(mapper(json_str), 'text')
-        self.assertEquals(mapper(json_str_or_null), 'text')
-        self.assertEquals(mapper(json_dt), 'timestamp_ntz')
-        self.assertEquals(mapper(json_dt_or_null), 'timestamp_ntz')
-        self.assertEquals(mapper(json_t), 'time')
-        self.assertEquals(mapper(json_t_or_null), 'time')
-        self.assertEquals(mapper(json_num), 'float')
-        self.assertEquals(mapper(json_int), 'number')
-        self.assertEquals(mapper(json_int_or_str), 'text')
-        self.assertEquals(mapper(json_bool), 'boolean')
-        self.assertEquals(mapper(json_obj), 'variant')
-        self.assertEquals(mapper(json_arr), 'variant')
+        self.assertEqual(mapper(json_str), 'text')
+        self.assertEqual(mapper(json_str_or_null), 'text')
+        self.assertEqual(mapper(json_dt), 'timestamp_ntz')
+        self.assertEqual(mapper(json_dt_or_null), 'timestamp_ntz')
+        self.assertEqual(mapper(json_t), 'time')
+        self.assertEqual(mapper(json_t_or_null), 'time')
+        self.assertEqual(mapper(json_num), 'float')
+        self.assertEqual(mapper(json_int), 'number')
+        self.assertEqual(mapper(json_int_or_str), 'text')
+        self.assertEqual(mapper(json_bool), 'boolean')
+        self.assertEqual(mapper(json_obj), 'variant')
+        self.assertEqual(mapper(json_arr), 'variant')
+        self.assertEqual(mapper(json_binary), 'binary')
+
+    def test_column_trans(self):
+        """Test column transformation"""
+        trans = db_sync.column_trans
+
+        # Incoming JSON schema types
+        json_str = {"type": ["string"]}
```

Comment:
Lines 61-73 seem to be repeating 95-107

Reply:
Those are different tests: `test_column_type_mapping` & `test_column_trans`.

Reply:
Yeah, but it's still repeating code, it's better to isolate it. We can put it in `setUp` for example, and then reuse it. For example, you can create a dictionary inside `setUp` like `{'str': {"type": "string"}, 'str_or_null': {"type": ["string", "null"]} ......}`. 

If you use the proposed idea above, we can also get rid of repetitive `assertEqual`. You can just define a mapping from json schema type to SF column type and do the loop instead of many `assertEqual`

Reply:
Fair enough

Reply:
I have updated these tests.